### DISCUSSION TITLE
Example 'Show live alert' does not work (Bootstrap 5.3)

### DISCUSSION THREAD

**Discussion by merlincom** [twbs/bootstrap#39493] [Q&A]
Hallo
As I understand it, the two lines also include Popper [https://getbootstrap.com/docs/5.3/getting-started/introduction/](https://getbootstrap.com/docs/5.3/getting-started/introduction/)
Nevertheless, the live alert button does not work for me (Firefox). No error in the console. 

> Include Bootstrap’s CSS and JS. Place the <link> tag in the <head> for our CSS, and the <script> tag for our JavaScript bundle (including Popper for positioning dropdowns, poppers, and tooltips) before the closing </body>

My HTML

```html
<!doctype html>
<html lang="en">
  <head>
    <meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Bootstrap demo</title>
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN" crossorigin="anonymous">
  </head>
  <body>
    <h1>Hello, world!</h1>
    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-C6RzsynM9kWDrMNeT87bh95OGNyZPhcTNXj1NW7RuBCsyN/o0jlpcV8Qyq46cDfL" crossorigin="anonymous"></script>
  
	<div id="liveAlertPlaceholder"></div>
	<button type="button" class="btn btn-primary" id="liveAlertBtn">Show live alert</button>
</body>
```

[https://getbootstrap.com/docs/5.3/components/alerts/](https://getbootstrap.com/docs/5.3/components/alerts/)
Live example
Click the button below to show an alert (hidden with inline styles to start), then dismiss (and destroy) it with the built-in close button.

Tanks for help.
Marcel

**Comment by merlincom** [comment]
If I insert the required part of snipped.js into the website, it works. But it does not work if I include the whole snipped.js in the website.
`<script  src="./../bootstrap/site/assets/js/snippets.js"></script>`
What is my mistake in thinking?

**Reply by julien-deramond** [reply]
Your HTML is correct: it creates the placeholder where the alerts will be displayed in `<div id="liveAlertPlaceholder"></div>` and the button does nothing by default.

Then you can copy the JS content from https://getbootstrap.com/docs/5.3/components/alerts/#live-example in your own JavaScript file in your project. There's no need to copy our entire `snippets.js` file in your environment.

I've created quickly how you can do that in https://codepen.io/julien-deramond/pen/LYqvwvN:
- the content of the HTML file is the same as you
- the content of the JS file is only the few lines found in https://getbootstrap.com/docs/5.3/components/alerts/#live-example

Hope it can help :)

**Comment by merlincom** [comment]
Thank you very much for your helpful advice. I have only just started with Bootstrap and am still stumbling occasionally. But the learning curve is fun
